sw/device/lib/base/freestanding/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenTitan Freestanding C Headers
+
+This subtree defines headers requred for a C freestanding implementation, as specified in S4.6 of the C11 standard.
+Said headers are implemented to the letter as described in respective sections of said standard.
```

Comment:
It would be useful to state that they are only compatible with the RISCV ILP32 psABI, and the GCC and Clang compilers.